.github/workflows/docker.yml

```diff
@@ -0,0 +1,19 @@
+name: Docker Image CI
```

Comment:
not needed

Reply:
Noted and read file name vs the other one, they had it separated, you saw what I said in discord 🤷